openlibrary/templates/work_search.html

```diff
@@ -1,4 +1,4 @@
-$def with (input, q_param, do_search, get_doc, fulltext_search, facet_fields)
+$def with (input, q_param, do_search, get_doc, fulltext_search, facet_fields, safe_mode)
```

Comment:
```suggestion
$def with (input, q_param, do_search, get_doc, fulltext_search, facet_fields)
```
`safe_mode` no longer needed after a small refactor.